Dockerfile.cli.dev

```diff
@@ -0,0 +1,8 @@
+ARG from
+FROM $from
+
+COPY ./dev-chain/cli /root/.mesg-cli
```

Comment:
can't we just generate all the commands in the docker file instead of copy the files. This way we don't have to store and maintain the files and we can still have a deterministic chain with mnemonic as args of the docker.

I'm afraid that we will have a lot of issues if we do some migration of the genesis

Reply:
That's could be possible but I think there will be some issue with the way the cosmos commands takes password: there is not password flag and the passwords are read directly from stdin.
I would like to do this (if possible) in another PR.

There will not be any migration of genesis as this dev version is intended to be used on temp volume that are discarded at every start/stop of the associated docker.

Reply:
The migration I'm talking about is if we add a new parameter and the structure of the genesis changes for some reasons. It would be better to re-generate it all the time or at least have tests to ensure that breaking changes on the genesis will be changed on the dev directory. But otherwise yes a script if possible would be nice in another PR but not a priority

Reply:
In this current version, if the genesis is not valid, then the e2e tests will breaks.